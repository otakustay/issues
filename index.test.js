import {test, expect} from 'vitest';

const print = () => {
    const message = `
export default function () {
    function Foo() {
    }

    return Foo;
}
`;
    return message;
}

test('issuse', () => {
    expect(print()).toMatchSnapshot();
});
