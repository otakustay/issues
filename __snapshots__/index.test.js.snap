// Vitest Snapshot v1

exports[`issuse 1`] = `
"
export default function () {
    function Foo() {
    }

    return Foo;
}
"
`;
